libs/services/designer-client-services/src/lib/connector.ts

```diff
@@ -83,6 +83,14 @@ export interface IConnectorService {
     nodeMetadata: any
   ): Promise<OpenAPIV2.SchemaObject>;
 
+  /**
+   * Gets the dynamic swagger url in manifest based operations.
+   * @arg {string} operationId - The operation id.'
+   * @arg {any} defintion - The swagger definition.
+   * @return {Promise<string>}
+   */
+  getDynamicSwaggerUrl(connectorId: string, operationId: string, dynamicOperationId: string, defintion: string): Promise<string>;
```

Comment:
Can't we implement this in azurefunction and azureapim service and based on the manifest property we can call those services... since we already have separate services for them